lms/djangoapps/bookmarks/views.py

```diff
@@ -166,6 +167,19 @@ def get_queryset(self):
 
         return api.get_bookmarks(user=self.request.user, course_key=course_key, serialized=False)
 
+    def paginate_queryset(self, queryset):
+        """ Override GenericAPIView.paginate_queryset for the purpose of eventing """
+        page = super(BookmarksListView, self).paginate_queryset(queryset)
+        tracker.emit(
+            'edx.course.bookmark.listed',
+            {
+                'bookmarks_count': page.paginator.count,
+                'page_size': self.request.QUERY_PARAMS.get('page_size', self.paginate_by),
```

Comment:
Does page size mean the requested page size or the number of items which are actually in the list? If the former can we not use this: https://github.com/tomchristie/django-rest-framework/blob/version-2.4.x/rest_framework/generics.py#L216 ?